test/util/save_util_linux.cc

```diff
@@ -18,13 +18,15 @@
 
 #include "test/util/save_util.h"
 
+#define ENOSYS_SYSCALL_LINUX -1
+
 namespace gvisor {
 namespace testing {
 
 void MaybeSave() {
   if (internal::ShouldSave()) {
     int orig_errno = errno;
-    syscall(SYS_create_module, nullptr, 0);
+    syscall(ENOSYS_SYSCALL_LINUX, nullptr, 0);
```

Comment:
This needs to be a valid syscall that is not used in any of the syscall tests. We use it to trigger saving the sentry state when this syscall is called. I think you could use `finit_module` for arm64.

Save/restore tests have not been migrated to github yet, this is the reason you don't see any references to it.

Reply:
Done.
Thanks for your description. :)